azure-pipelines/build/templates/doclient-lite-docker-steps.yml

```diff
@@ -2,18 +2,30 @@
 # Consume this steps template in one or more jobs by passing in parameter values.
 
 parameters:
-- name: targetOsArch    # example: debian10-arm32
-  type: string
-- name: dockerImageName # example: jimsonmsft/debian10-arm32:latest
+- name: targetOsArch    # example: ubuntu1804-x64
   type: string
 - name: config          # debug/release/minsizerel/relwithdebuginfo
   type: string
 
 steps:
+- task: Docker@2
+  displayName: Login to ACR
+  inputs:
+    command: login
+    containerRegistry: doclientcontainerregistry
+    repository: $(parameters.targetOsArch)
```

Comment:
>$(parameters.targetOsArch) [](http://example.com/codeflow?start=16&length=26)

You will need to rename the repositories on azure portal now to change from dash to underscore.

Reply:
The github release yaml is still using dash (ubuntu1804-x64).

IIRC, pipeline job names do not accept dash which is the reason we had the OS and arch as separate and used dash/underscore as appropriate.

Reply:
Correct, saw the dash issue in pipeline job names as well. Let's use the underscore convention everywhere. 

Re: repositories on azure portal - do you mean the container repository?

Reply:
renamed to all use underscore, and just use a single parameter targetOsArch:
[https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/22a3997b-5698-4d18-b7f5-0d2b6af877bf/resourceGroups/do-client/providers/Microsoft.ContainerRegistry/registries/DOClientContainerRegistry/repository](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/22a3997b-5698-4d18-b7f5-0d2b6af877bf/resourceGroups/do-client/providers/Microsoft.ContainerRegistry/registries/DOClientContainerRegistry/repository)